### ISSUE TITLE
Blogpost and tutorial about how to use DeepAR with Ant Media Server JS SDK 

### ISSUE THREAD

**Issue by mekya** [ant-media/Ant-Media-Server#4802]

**Comment by timantmedia** [comment]
Post complete https://antmedia.io/wp-admin/post.php?post=70780&action=elementor for @timantmedia to review

**Comment by mekya** [comment]
Hey @timantmedia ,

Is there anything you expect from the team or this is just giving the information?

Regards

**Comment by timantmedia** [comment]
@mekya No, I was tagging myself for a reminder! I'll read it today and publish it.

**Comment by timantmedia** [comment]
@USAMAWIZARD I made some edits and format changes. Please confirm you are fine with it https://antmedia.io/?p=70780&preview_id=70780&preview_nonce=383ddc75f7&preview=true

**Comment by timantmedia** [comment]
Is this a duplicate of #4718 ?

**Comment by USAMAWIZARD** [comment]
yes its duplicate

**Comment by USAMAWIZARD** [comment]
Yes+
